### ISSUE TITLE
Add exclusions

### ISSUE THREAD

**Issue by s-spindler** [sonatype-nexus-community/nancy#16]
I would like to add nancy to the build of a Go project and have it fail if a vulnerability is discovered.
To do this I would need an option to exclude certain vulnerabilities from the scan as there might be false positives and vulnerabilities that do not apply in a certain context.
This would be similar to e.g. the ossindex maven plugin: https://sonatype.github.io/ossindex-maven/maven-plugin/excludes.

**Comment by DarthHater** [comment]
@s-spindler ! Thanks for filing this. How would you imagine excluding them? A list that is picked up?

**Comment by s-spindler** [comment]
@DarthHater Yes exactly. Like in the maven plugin it could be a list of index IDs to exclude single vulnerabilities. I'm not a fan of excluding vulnerabilities on a dependency level so I have no use for exclusion by coordinates but I guess this could also be added.

**Comment by DarthHater** [comment]
@s-spindler most of the work for this is done in #28 now if you want to test drive it!
